### PR TITLE
Make clickhouse-server provides keeper

### PR DESCRIPTION
**Pull request by Felixoid**
### Changelog category (leave one):
- Build/Testing/Packaging Improvement

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
Solve issue with launching standalone clickhouse-keeper from clickhouse-server package.

### Documentation entry for user-facing changes

- [ ] Documentation is written (mandatory for new features)

> Information about CI checks: https://clickhouse.com/docs/en/development/continuous-integration/

### CODE REVIEW

**Comment by robot-ch-test-poll1** [comment]
*This is an automated comment for commit a4144b4fda3b559626af63751e49429b120a5a54 with description of existing statuses. It's updated for the latest CI running*

[⏳ Click here](https://s3.amazonaws.com/clickhouse-test-reports/55226/a4144b4fda3b559626af63751e49429b120a5a54/ci_running.html) to open a full report in a separate page

<details><summary>Successful checks</summary>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>Mergeable Check</td><td>Checks if all other necessary checks are successful</td><td><a href="https://github.com/ClickHouse/ClickHouse/actions/runs/6404138605">✅ success</a></td></tr>
<tbody>
</table>
</details>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>CI running</td><td>A meta-check that indicates the running CI. Normally, it's in <b>success</b> or <b>pending</b> state. The failed status indicates some problems with the PR</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/55226/a4144b4fda3b559626af63751e49429b120a5a54/ci_running.html">⏳ pending</a></td></tr>
<tbody>
</table>

**Comment by Felixoid** [comment]
@den-crane I'd appreciate if you can confirm that it solve the issue we discussed

File: packages/clickhouse-server.yaml [hunk 1]
```diff
@@ -9,24 +16,21 @@ homepage: "https://clickhouse.com"
...
+    Source: clickhouse
 
-conflicts:
-- clickhouse-keeper
+# Package specific content
 replaces:
 - clickhouse-server-common
 - clickhouse-server-base
 provides:
+- clickhouse-keeper
```
Comment (alexey-milovidov) [inline packages/clickhouse-server.yaml:29]: This is the relevant change.

PS. I didn't understand everything else in the diff, but it looks good to me.

**Review by alexey-milovidov — APPROVED** [review]

**Comment by Felixoid** [comment]
Added the service and config files as well

**Pull request merged**